Gruntfile.js

```diff
@@ -53,7 +53,7 @@ module.exports = function(grunt) {
       grunt.log.writeln('task skipped because this version of Node is too old');
```

Comment:
ooh, I just noticed that this function needs updating while you're working on it. We [no longer support](https://github.com/mozilla/web-ext/blob/master/.travis.yml#L3-L5) this old Node version so the if branch that uses `semver` can be deleted.